src/intersectionWith.js

```diff
@@ -12,7 +12,7 @@ var uniqWith = require('./uniqWith');
  * @memberOf R
  * @since v0.1.0
  * @category Relation
- * @sig (a -> a -> Boolean) -> [*] -> [*] -> [*]
+ * @sig ((a, a) -> Boolean) -> [a] -> [a] -> [a]
```

Comment:
I think that is a fair characterization. I also don't wanna get bogged down on this, but I don't wanna just steamroll @davidchambers either. I prefer the "set" semantics, but if David has a strong argument in favor the "practical" side I wanna hear it.